terraform/.terraform.lock.hcl

```diff
@@ -1,38 +0,0 @@
-# This file is maintained automatically by "terraform init".
```

Comment:
Err, why did you delete the infrastructure lock file?

Reply:
fixed it.